### PR TITLE
feat(vscode): support deno-aleph

### PR DESCRIPTION
**Pull request by kecrily**
https://github.com/alephjs/aleph.js/blob/main/server/types.ts

close https://github.com/unocss/unocss/issues/901

### CODE REVIEW

File: packages/vscode/src/index.ts [hunk 1]
```diff
@@ -35,6 +35,10 @@ export async function activate(ext: ExtensionContext) {
         files: 'nuxt.config',
         fields: 'unocss',
       }),
+      sourceObjectFields({
+        files: 'aleph.config',
+        fields: 'atomicCSS',
```
Comment (ije) [inline packages/vscode/src/index.ts:40]: i am considering to rename `atomicCSS ` to `unocss`

**Comment by ije** [comment]
thanks for supporting aleph.js, unocss is great project!

File: packages/vscode/src/index.ts (hunk 1 above)
Comment (antfu) [reply packages/vscode/src/index.ts:40]: Yeah I think `unocss` is better for explictness 👍

File: packages/vscode/package.json [hunk 2]
```diff
@@ -39,6 +39,7 @@
     "workspaceContains:**/nuxt.config.*",
     "workspaceContains:**/uno.config.*",
     "workspaceContains:**/unocss.config.*",
+    "workspaceContains:**/aleph.config.*",
```
[the remark this answers is not in the document]
Comment (kecrily) [reply packages/vscode/package.json:42]: I saw it from here, it's probably too old. I will update it
https://alephjs.vercel.app/docs/api-reference/config

[the remark this answers is not in the document]
Comment (ije) [reply packages/vscode/package.json:42]: > I saw it from here, it's probably too old. I will update it https://alephjs.vercel.app/docs/api-reference/config

thanks!

File: packages/vscode/src/index.ts [hunk 3]
```diff
@@ -35,6 +35,16 @@ export async function activate(ext: ExtensionContext) {
...
       }),
+      sourceObjectFields({
+        files: 'aleph.config',
+        extensions: ['js', 'mjs', 'jsx', 'ts', 'tsx'],
+        fields: 'serve/config/unocss',
+      }),
+      sourceObjectFields({
+        files: 'deno',
+        extensions: ['json', 'jsonc'],
+        fields: 'serve/config/unocss',
```
Comment (antfu) [inline packages/vscode/src/index.ts:46]: Those fields looks a bit strange, what's the configure file would look like?

Comment (kecrily) [reply packages/vscode/src/index.ts:46]: I referenced this
https://github.com/alephjs/aleph.js/blob/main/server/helpers.ts

**Comment by tomkimsour** [comment]
> https://github.com/alephjs/aleph.js/blob/main/server/types.ts
> 
> close #901

Should I close it ? the merge above didn't fully work.

**Comment by ije** [comment]
i think this pr is not necessary, to enable vsc extension, we can import `unocss.confg.js` in `server.tsx`

**Pull request closed without merging**